### PR TITLE
Change app name

### PR DESCRIPTION
**Pull request by boniattirodrigo**

### CODE REVIEW

**Comment by PedroFelipe** [comment]
My suggestion to improve these translations:

iOS
```Open Xcode and select ReactNativeStarterKit as your target, open General tab and type your app name on Display Name field.

<img width="828" alt="screen shot 2017-07-10 at 09 13 17" src="https://user-images.githubusercontent.com/463661/28017570-425e3f2c-6550-11e7-8d9c-a6f9ad17d170.png">
```

I noticed that by default a target may or not be selected, so there's an updated screenshot to illustrate it. What do you think?

Android
```
Update `app_name` value on `android/app/src/main/res/values/strings.xml` file.
```

File: change-app-name.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Change app name
+
+## iOS
+Open Xcode and select ReactNativeStarterKit as your target, click on General tab and type your app name on Display Name field.
+
+![](assets/images/illustrations/change-app-name.png)
+
+By default a target may or not be selected, if not you just need to select it.
```
Comment (willamesoares) [inline change-app-name.md]: Can you rephrase this to `may or may not be` ?
